lib/resolver/live-resolver-query.js

```diff
@@ -1,4 +1,9 @@
 export default function ({ type, target }) {
+  if (type == 'npm') {
+    // If the target looks like 'foo/bar.js', pretend it is 'foo' instead. See
+    // https://github.com/OctoLinker/browser-extension/issues/93
+    target = target.split('/')[0];
+  }
```

Comment:
Hmm, yeah, I also wasn't completely sure where it should go. My logic at the time was to put it as close to the server as possible, in case we wanted to move the logic into live-resolver in order to account for the edge cases mentioned in https://github.com/OctoLinker/live-resolver/pull/3#issuecomment-141370198

However, I don't yet understand why it should be a separate resolver. To me, that seems like an unnecessary layer of complexity, but perhaps there are some cases I haven't considered that would be broken by changing the `liveResolverQuery` directly.

At any rate, I've updated the PR with some changes inspired by your snippet above, as well as a test. Let me know what you think!